gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java

```diff
@@ -33,6 +33,15 @@
  */
 @InterfaceStability.Unstable
 public enum GhfsStatistic {
+  /** Http Request counters */
+  ACTION_HTTP_HEAD_REQUEST(
+      StoreStatisticNames.ACTION_HTTP_HEAD_REQUEST, "HEAD request.", TYPE_COUNTER),
+  ACTION_HTTP_GET_REQUEST(
+      StoreStatisticNames.ACTION_HTTP_GET_REQUEST, "GET request.", TYPE_COUNTER),
+  ACTION_HTTP_HEAD_REQUEST_FAILURES(
+      "action_http_head_request_failures", "HEAD request.", TYPE_COUNTER),
+  ACTION_HTTP_GET_REQUEST_FAILURES(
+      "action_http_get_request_failures", "GET request.", TYPE_COUNTER),
```

Comment:
Let's use the prefix from `StoreStatisticNames` enum

Reply:
Updated